graphql/schemabuilder/pagination.go

```diff
@@ -51,14 +51,31 @@ type PaginationArgs struct {
 	Before *string
 }
 
+func (p PaginationArgs) Limit() int {
```

Comment:
should this be private api?

Reply:
Yes, thanks. I tend to prefer uppercase on internal stuff to indicate that it's being used elsewhere, but in this case it's an exported struct. 👍